ledger/simulation/tracer.go

```diff
@@ -185,12 +185,14 @@ func (tracer *evalTracer) AfterTxnGroup(ep *logic.EvalParams, deltas *ledgercore
 	}
 }
 
-func (tracer *evalTracer) saveApplyData(applyData transactions.ApplyData) {
+func (tracer *evalTracer) saveApplyData(applyData transactions.ApplyData, omitEvalDelta bool) {
 	applyDataOfCurrentTxn := tracer.mustGetApplyDataAtPath(tracer.absolutePath())
 	// Copy everything except the EvalDelta, since that has been kept up-to-date after every op
 	evalDelta := applyDataOfCurrentTxn.EvalDelta
 	*applyDataOfCurrentTxn = applyData
-	applyDataOfCurrentTxn.EvalDelta = evalDelta
+	if omitEvalDelta {
```

Comment:
"omit" means skip, right? why is it assigned in this case? Like the condition should be reversed?

Reply:
Good question. The `evalDelta` variable is the old data, so if `omitEvalDelta` is true, we keep the old data in `applyDataOfCurrentTxn`. If false then we take the new data from `applyData`.

Reply:
Added comment in 8fd295f56